google/ggrc-core
# GGRC-4060 Global Creator as Program Reader and Global Reader can Raise Issue in Assessment where they have readonly rights

# Issue description

Global Creator as Program Reader and Global Reader should not Raise Issue in Assessment where they have readonly rights

# Steps to test the changes

1. Assign Global Creator 'Program Reader' role in a program
3. Create an Audit and go to the audit page
4. Create Assessment
5. Log as Global Creator and open audit page
6. Navigate to Assessment Info panel > Raise Issue tab
7. Expected: Rase Issue button doesn't displays

# Solution description

Changed add-issue-button render's condition.

# Sanity checklist




# PR Review checklist
